docs/quick_start_python.rst

```diff
@@ -18,13 +45,14 @@ defined from within source code for a specific point and density.
         computation_point = ... # [] or np.array of length 3
 
         # Evaluate the gravity model
-        # Notice that the last argument could also be a list of points
-        # Returns (list of) tuple of potential, acceleration and tensor
-        potential, acceleration, tensor = model.evaluate(vertices, faces, density, computation_point)
+        # Notice that the third argument could also be a list of points
+        # Returns a tuple of potential, acceleration and tensor
+        potential, acceleration, tensor = model.evaluate((vertices, faces), density, computation_point, parallel=True)
```

Comment:
named arguments? :)

Reply:
Yes, I use both notations, so the reader can basically chose the flavour :)